drivers/hmis/app/models/hmis/filter/enrollment_filter.rb

```diff
@@ -49,4 +50,52 @@ def with_project_types(scope)
   def with_search_term(scope)
     with_filter(scope, :search_term) { scope.matching_search_term(input.search_term) }
   end
+
+  def with_household_tasks(scope)
+    with_filter(scope, :household_tasks) do
+      return scope.all unless input.household_tasks&.include?('ANNUAL_DUE')
+
+      start_date = Arel.sql <<~SQL
+        make_date(
+          extract(year from current_date)::integer,
+          extract(month from "EntryDate")::integer,
+          extract(day from "EntryDate")::integer
+        ) - interval '30 days'
+      SQL
+
+      last_start_date = Arel.sql <<~SQL
+        make_date(
+          extract(year from current_date)::integer,
+          extract(month from "EntryDate")::integer,
+          extract(day from "EntryDate")::integer
+        ) - interval '30 days' - interval '1 year'
+      SQL
+
+      # SQL-ized version of the logic here: drivers/hmis/app/models/hmis/reminders/reminder_generator.rb#annual_assessment_reminder
+      scope = scope.
+        joins(:household).
+        joins(
+          e_t.
+          join(cas_t, Arel::Nodes::OuterJoin).
+          on(
+            cas_t[:enrollment_id].eq(e_t[:enrollment_id]).
+            and(cas_t[:data_collection_stage]).eq(5).
+            and(cas_t[:wip]).eq(false),
+          ).join_sources,
+        ).
+        # Earliest entry was more than a year ago
+        where(hh_t[:earliest_entry].lteq(Date.today - 1.year)).
+        # Household is not exited
+        where(hh_t[:latest_exit].eq(nil)).
+        where(
+          # There haven't been any annual assessments yet
```

Comment:
```suggestion
          # This enrollment has not had any annual assessments
```
Maybe clearer? I was confused, at first I thought that this meant there hadn't been any annuals in the _household_.